### ISSUE TITLE
Fix TJPCov GitHub actions

### ISSUE THREAD

**Issue by carlosggarcia** [LSSTDESC/SprintWeek2024#1]
# Fix TJPCov GitHub actions

The GitHub actions (and maybe the code) are broken. Probably due to the new version of NaMaster.

Contacts: @carlosggarcia 
Day/Time: Tuesday 29, 9-12am UK time
Main communication channel: Slack
GitHub repo: https://github.com/LSSTDESC/TJPCov
Zoom room (if applicable):

## Goals and deliverable

Make TJPCov work again

## Resources and skills needed

Python and some knowledge of TJPCov and NaMaster

## Detailed description

Not much more to add.

**Comment by arunkannawadi** [comment]
Do you think this is new contributor friendly as well?

**Comment by carlosggarcia** [comment]
Yes, I think this one would be a good exercise for new contributors

**Comment by arunkannawadi** [comment]
@paulrogozenski is running a tutorial involving TJPCov on Tuesday morning (10am-12pm PT). It'd be great if this sprint could happen after that.

**Comment by carlosggarcia** [comment]
I've updated the schedule. I won't be able to make it on Tuesday after the TJPCov tutorial but I think people should go ahead and try to fix this. Otherwise, I will do it on Wednesday UK morning.

**Comment by carlosggarcia** [comment]
Update: I will work on this issue today at 4pm (9 PT).

**Comment by carlosggarcia** [comment]
WIP in https://github.com/LSSTDESC/TJPCov/pull/112

So far I've solved the issues with NaMaster v2. There is a problem with the binning implementation that was introduced in a recent PR.
